docs/shared/common_prereqs/common_prereqs.adoc

```diff
@@ -0,0 +1,181 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2020, 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+//Contains content that is shared between multiple guides.
+:common-page-prefix-inc:
+
+
+=== Prerequisites 
+
+// tag::common-prereqs[]
+
+
+[role="flex, sm7"]
+|=======
+|A Helidon MP Application | You can use your own application or use the https://helidon.io/docs/v2/#/mp/guides/02_quickstart[Helidon MP Quickstart] to create a sample application.
+|https://www.oracle.com/technetwork/java/javase/downloads[Java{nbsp}SE{nbsp}11] (http://jdk.java.net[Open{nbsp}JDK{nbsp}11]) |Helidon requires Java 11+.
+|https://maven.apache.org/download.cgi[Maven 3.6.1+]|Helidon requires Maven 3.6.1+.
+|https://docs.docker.com/install/[Docker 18.09+]|You need Docker if you
+want to build and deploy Docker containers.
+|https://kubernetes.io/docs/tasks/tools/install-kubectl/[Kubectl 1.16.5+]|If you want to
+deploy to Kubernetes, you need `kubectl` and a Kubernetes cluster (you can
+<<about/05_kubernetes.adoc,install one on your desktop>>).
+|=======
+
+
+
+[source,bash]
+.Verify Prerequisites
+----
+java -version
+mvn --version
+docker --version
+kubectl version --short
+----
+
+
+[source,bash]
+.Setting JAVA_HOME
+----
+# On Mac
+export JAVA_HOME=`/usr/libexec/java_home -v 11`
+
+# On Linux
+# Use the appropriate path to your JDK
+export JAVA_HOME=/usr/lib/jvm/jdk-11
+----
+// end::common-prereqs[]
+
+
+// tag::common-prereqs-helm[]
+
+[role="flex, sm7"]
+|=======
+|A Helidon MP Application | You can use your own application or use the https://helidon.io/docs/v2/#/mp/guides/02_quickstart[Helidon MP Quickstart] to create a sample application.
```

Comment:
For future reference: It looks as if an `.adoc` file tag can `include` another tag from the same file. This would let us avoid repeating the same content in each of the tags in this common file which have the shared prereqs plus some additional one(s). The Helm, curl, etc. tag sections could `include::common_prereqs.adoc[tag=common-prereqs]` and then add their respective specific additional pre-reqs.